File: Android/SoonZik/app/src/main/java/com/soonzik/soonzik/Description.java
package com.soonzik.soonzik;

import org.json.JSONObject;

/**
 * Created by kevin_000 on 16/06/2015.
 */
public class Description extends ActiveRecord {

    private int id = -1;
    private String description = "";
    private String language = "";

    public Description() {}

    public Description(JSONObject json) {
        super.createInstance(this, json, this.getClass());
    }

    @Override
    public String toString() {
        String str = "id = " + Integer.toString(id)
                + " : description = " + description
                + " : language = " + language;

        return (str);
    }

    public String getLanguage() {
        return language;
    }

    public String getDescription() {
        return description;
    }
}

File: iOS SoonZik/SoonZik/Models/Description.m
//
//  Description.m
//  SoonZik
//
//  Created by Maxime Sauvage on 27/06/15.
//  Copyright (c) 2015 SoonZik - Maxime SAUVAGE. All rights reserved.
//

#import "Description.h"

@implementation Description

- (id) initWithJsonObject:(NSDictionary *)json {
    self = [super init];
    
    self.identifier = [[json objectForKey:@"id"] intValue];
    self.text = [json objectForKey:@"description"];
    self.language = [json objectForKey:@"language"];
    
    return self;
}

@end
